capstone-frontend/src/components/UserCard.jsx

```diff
@@ -7,36 +7,78 @@ import '../styles/Groups.css';
 class UserCard extends Component {
   removeMember = () => {
     const jsonBody = {
-      "remove_memberIDs": this.props.user.id, 
+      "remove_memberIDs": this.props.user.id,
```

Comment:
nit: Remove the quotes around the field names.

Reply:
Will do